crates/uv/tests/pip_sync.rs

```diff
@@ -4159,12 +4286,20 @@ fn require_hashes_find_links_invalid_hash() -> Result<()> {
         .arg("--find-links")
         .arg("https://raw.githubusercontent.com/astral-test/astral-test-hash/main/invalid-hash/simple-html/example-a-961b4c22/index.html"), @r###"
     success: false
-    exit_code: 1
+    exit_code: 2
     ----- stdout -----
 
     ----- stderr -----
-      × No solution found when resolving dependencies:
-      ╰─▶ Because example-a-961b4c22==1.0.0 is unusable because the hash does not match and you require example-a-961b4c22==1.0.0, we can conclude that the requirements are unsatisfiable.
+    Resolved 1 package in [TIME]
+    error: Failed to download distributions
+      Caused by: Failed to fetch wheel: example-a-961b4c22==1.0.0
+      Caused by: Hash mismatch for example-a-961b4c22==1.0.0
+
+    Expected:
+      sha256:123
+
+    Computed:
+      sha256:5d69f0b590514103234f0c3526563856f04d044d8d0ea1073a843ae429b3187e
```

Comment:
These aren't as nice-looking, but keep in mind that for `--require-hashes`, the user has to pin _all_ dependencies upfront anyway. There's no backtracking or anything.

Reply:
They're pretty ugly but 🤷‍♂️ we could address later if we want.

Reply:
Absolutely DESTROYING my error messages